docs/src/pages/demos/paper/PaperSheet.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Theme, createStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/styles';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      ...theme.mixins.gutters(),
```

Comment:
Could you replace the gutter usage with theme.spacing()?

Reply:
This is only porting the demo from JS to TS. Any behavioral changes should be discussed independently.

Reply:
I agree, our top priority should be about the TypeScript migration. We can handle this topic latter on.

Reply:
reveted back